src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicTypeInferenceService.TypeInferrer.vb

```diff
@@ -838,7 +851,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
 
                 Dim index = argumentList.Arguments.GetWithSeparators().IndexOf(previousToken)
-                Return (index + 1) \ 2
```

Comment:
the corresponding C# implementation also has does the same calculation with index. Is it worth checking that this boundary case is covered over there as well?